editor/src/clj/editor/attachment.clj

```diff
@@ -79,10 +81,13 @@
                        additions; a coll of 2-element tuples where first element
                        is a list-kw keyword, and second is a map with the
                        following keys:
-                         :init    required, data structure that will be supplied
-                                  to the init-fn to create element
-                                  initialization transaction steps
-                         :add     optional, attachment tree of the created item
+                         :init         required, data structure that will be
+                                       supplied to the init-fn to create element
+                                       initialization transaction steps
+                         :node-type    required, node type of child node
```

Comment:
Now we require a node type here. Editor scripts, though, only require a node type when there is more than 1 alternative type registered.